src/plugin/logger.js

```diff
@@ -45,7 +46,14 @@ class Logger extends Plugin {
       const toLog = cloneDeep({ event, response });
       this.parse(toLog);
       this.redactor(toLog);
-      logger[success ? 'info' : 'warn'](JSON.stringify(toLog));
+      const prefix = [
+        get(toLog, 'response.statusCode'),
+        get(toLog, 'event.httpMethod'),
+        get(toLog, 'event.path')
```

Comment:
Is there a way we can expose the `route` here?
`400: GET /offers/id1` !==  `400: GET /offers/id2`